support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -115,4 +115,23 @@ export const tests: Tests = {
 		seed: 10,
 		optimizeId: 'dQCXBc3QQIW7M1Di_qSCHw',
 	},
+	singleToRecurring: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'variant',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: false,
+		referrerControlled: false,
+		seed: 15,
```

Comment:
I think we should add a `targetPage: pageUrlRegexes.contributions.allLandingPagesAndThankyouPages` to this AB test, this will mean we only assign users on the "contributions" checkout to this AB test (which is what we want)

Reply:
Ah yes, have done and will use this going forwards to reduce the pages affected by ab-tests.